scripts/remove

```diff
@@ -10,10 +10,13 @@ source _common.sh
 source /usr/share/yunohost/helpers
 
 #=================================================
-# RUNNER REGISTRATION
+# RUNNER UNREGISTRATION
 #=================================================
+ynh_script_progression --message="Unregistering the runner on Peertube server..." --weight=1
 
-peertube-runner unregister --url runner_url --registration-token runner_token --runner-name $app$(hostname)
+runner_id = ynh_app_setting_get --app=$app --key=runner_id
```

Comment:
You don't need to fetch the runner_id setting, this is app v2 format, all settings are automatically available in scripts